app/config/locales.inc.php

```diff
@@ -32,7 +32,7 @@
 
 // List of locales only working on mozilla.org
 $mozorg_locales = [
-    'am', 'ncj', 'nv', 'zam',
```

Comment:
Given that the PR was already reviewed, I would have used a different PR.

At this point, keep it here, but note that you will have to a bunch of files on mozilla.org **and** engagement-l10n